### PR TITLE
Custom Code: Fixed - empty state, display link to trashed items(ED-1989)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: includes/template-library/sources/local.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
 use Elementor\Modules\LandingPages\Module as Landing_Page_Module;
 use Elementor\Modules\Library\Documents\Library_Document;
 use Elementor\Plugin;
+use ElementorPro\Modules\CustomCode\Module;
```
Comment (matipojo) [inline includes/template-library/sources/local.php:12]: Remove the pro code.
Make it generic for all types.

File: includes/template-library/sources/local.php [hunk 2]
```diff
@@ -1357,6 +1358,17 @@ public function print_blank_state_template( $current_type_label, $href, $descrip
 					printf( __( 'Add New %s', 'elementor' ), $current_type_label );
 					?>
 				</a>
+				<?php
+				$module_name = str_replace( '_', ' ', module::MODULE_NAME );
```
Comment (matipojo) [inline includes/template-library/sources/local.php:1362]: Never use labels, they may be translated. try to find the type via global `$typenow` or via `get_current_screen()`.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/template-library/sources/local.php (hunk 1 above)
Comment (etayf) [reply includes/template-library/sources/local.php:12]: Fixed

File: includes/template-library/sources/local.php (hunk 2 above)
Comment (etayf) [reply includes/template-library/sources/local.php:1362]: Fixed

File: includes/template-library/sources/local.php [hunk 3]
```diff
@@ -1357,6 +1357,17 @@ public function print_blank_state_template( $current_type_label, $href, $descrip
 					printf( __( 'Add New %s', 'elementor' ), $current_type_label );
 					?>
 				</a>
+				<?php
+				$post_type = get_current_screen()->post_type;
+				if ( isset( $post_type ) ) :
```
Comment (matipojo) [inline includes/template-library/sources/local.php:1362]: How can it be not set? (or empty)

File: includes/template-library/sources/local.php [hunk 4]
```diff
@@ -1357,6 +1357,17 @@ public function print_blank_state_template( $current_type_label, $href, $descrip
...
+					$trashed_posts_query = new \WP_Query( [
+						'post_type' => $post_type,
+						'post_status' => 'trash',
+					] );
+					if ( $trashed_posts_query->post_count > 0 ) : ?>
+						<p><?php echo __( 'Or view', 'elementor' ); ?> <a href='./edit.php?post_status=trash&post_type=<?php echo $post_type; ?>'><?php echo __( 'Trashed Items', 'elementor' ); ?></a></p>
```
Comment (matipojo) [inline includes/template-library/sources/local.php:1368]: Use `admin_url()`.
Break lines between the HTML tags and their content.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**